ssn/frag/Origins.html

```diff
@@ -1,119 +1,75 @@
-            even more restrained axiomatization to foster wide reuse and adaptation among an audience that is not
-            necessarily familiar with OWL. SOSA makes use of the <code>domainIncludes</code> and
-            <code>rangeIncludes</code> annotation properties defined in Schema.org. These had not been available before.
-          </li>
-          <li>Given the increased interest in using Semantic Web technologies directly on the level of individual
-            sensors, actuators, or platforms, SOSA's axiomatization does not use many of the more complex language
-            elements introduced by SSN.</li>
-        </ul>
-    </ul>
-<!--  </section>  -->
+<!-- <section id="origins"> -->
+
+<h3>Origins of SSN and SOSA</h3>
+
+<h4>OGC Sensor Web Enablement</h4>
+
+<p>
+    Starting in 2002, the OGC's Sensor Web Enablement initiative [[SWE]] developed a generic framework for delivering
+    sensor data, dealing with remote-sensing, moving platforms, and in-situ monitoring and sensing.
+    In particular, the key standards SensorML and O&amp;M provided complementary viewpoints:</p>
+<ol>
+    <li>
+        <b>SensorML</b> [[SensorML]] is 'provider-centric' and encodes details of the sensor along with raw observation
+        data
+    </li>
+    <li>
+        Observations and Measurements (<b>O&amp;M</b>) [[OandM]] [[iso-19156-2011]] was designed to be more
+        'user-centric', with the target of the observation and the observed property as first-class objects.
+    </li>
+</ol>
+<p>
+    O&amp;M also provided a model for sampling, since almost all scientific observations are made on a subset of, or
+    proxy for, the ultimate feature of interest.
+</p>
+
+<h4>W3C Semantic Sensor Network Incubator</h4>
+
+<p>
+    The initial Semantic Sensor Network ontology - now known as <b>SSNX</b> [[SSNX]] - was developed by the <a
+        href="https://www.w3.org/2005/Incubator/ssn/wiki/Main_Page">W3C Semantic Sensor Network Incubator Group</a>.
+    SSNX was influenced by SWE, but was primarily built around the <em>Stimulus Sensor Observation</em> (<b>SSO</b>)
+    ontology design pattern [[SSO-Pattern]].
+    The SSO was also aligned to the <em>Dolce-Ultralite</em> upper ontology [[DUL]].
+</p>
+
+<h4>SSN Ontology, including SOSA</h4>
+
+<p>
+    The <b>SSN Ontology</b> [[vocab-ssn-20171019]] combined key elements of SWE and SSNX, and added Actuation and
+    Sampling as additional procedure execution types.
+    The core classes and properties comprised the <em>Sensor, Observation, Sample, and Actuator</em> (<b>SOSA</b>)
+    module.
+    SOSA acts as the central building block for the SSN and is packaged with very lightweight axiomatization so as to
+    support use by web-developers.
+</p>
+
+<p>For details on the design philosophy and decisions around the SSN Ontology, please consult chapter "3. Origins of
+    SSN and SOSA" in [[[vocab-ssn-20171019]]].
+</p>
+
+<h4>New edition</h4>
```

Comment:
suggestion to move it plural "New edition**s**'  

as we speak about both OMS and SSN Ontology new editions

Reply:
Accepted